app/api/resources/user.py

```diff
@@ -15,13 +15,63 @@
 from app.api.email_utils import send_email_verification_message
 from app.api.models.user import *
 from app.api.dao.user import UserDAO
+from app.database.models.user import UserModel
 from app.api.resources.common import auth_header_parser
+from google.oauth2 import id_token
+from google.auth.transport import requests
+import os
 
 users_ns = Namespace("Users", description="Operations related to users")
 add_models_to_namespace(users_ns)
 
 DAO = UserDAO()  # User data access object
 
+def create_tokens_for_new_user_and_return(user):
+    # create tokens and expiry timestamps
+    access_token = create_access_token(identity=user.id)
+    refresh_token = create_refresh_token(identity=user.id)
+
+    from run import application
+    access_expiry = datetime.utcnow() + application.config.get(
+        "JWT_ACCESS_TOKEN_EXPIRES"
+    )
+    refresh_expiry = datetime.utcnow() + application.config.get(
+        "JWT_REFRESH_TOKEN_EXPIRES"
+    )
+
+    # return data
```

Comment:
```suggestion
```
Is this line for debugging purposes?

Reply:
No, this is just a comment, shall I remove it?